src/work-allocation/containers/task-manager-list/task-manager-list.component.ts

```diff
@@ -124,4 +127,17 @@ export class TaskManagerListComponent extends TaskListWrapperComponent implement
     }
     return { key: 'assignee', operator: 'IN', values };
   }
+
```

Comment:
This should be removed. I already did so, which means there's a merge issue that's reintroduced it.

Reply:
merge issue, will resolve